src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -41,6 +41,7 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = ({
     false
   )
   const [stagedSecondFactor, setStagedSecondFactor] = useState(null)
+  const [confirmedPassword, setConfirmedPassword] = useState("")
```

Comment:
Do I need to include `setConfirmedPassword("")` in the `resetState`? It appears that we do this for all _state flags_ in [AppSecondFactor](https://github.com/artsy/force/blob/main/src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx#L49) however I am not clear on whether this is necessary seeing as in [SmsSecondFactor](https://github.com/artsy/force/blob/main/src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/SmsSecondFactor/index.tsx#L50) not _all_ the flags are 'reset'.

Reply:
Wondering, what is the reason for the reset?

Reply:
Based on the [note](https://github.com/artsy/force/blob/main/src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/SmsSecondFactor/index.tsx#L49) from @dzucconi, looks like we shouldn't need to worry about this?

Maybe we were seeing some stale state and were testing resetting them manually for some reason? Looks like this only is being done after a `mutation` [error](https://github.com/artsy/force/blob/0fa8a5e244e1eb0856957ff63d433dd96cb4e48c/src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx#L116). Not sure if you saw anything while testing a mutation failure?

Reply:
> You may need to add it. The whole mess was getting into unrecoverable states on error, so I added that `resetState` to kind of take a sledgehammer to it.

Thanks for elaborating. For consistency, I'll add it in.